modules/cudf/test/cudf-data-frame-tests.ts

```diff
@@ -832,3 +856,8 @@ test(`DataFrame.dropDuplicates("first", true, true, ['a'])`, () => {
   expect([...result.get('a')]).toEqual([null, 1, 3, 4]);
   expect([...result.get('b')]).toEqual([null, 5, 8, 2]);
 });
+
+// Typescript does not allow us to throw a compile-time error if
+// the result of a method is `never`. Instead, let's just verify
+// the result is `never`.
+function _verifySeriesIsNever(_: never) { return _; }
```

Comment:
shouldn't this be `_verifySeriesIsNever(_: Series<never>)` ?

Reply:
I need to double check the difference, but technically what I return is just `never`, whereas this https://github.com/rapidsai/node-rapids/pull/240 has `Dataframe<never>`. Should check out the difference.

Reply:
I was referring to the result of `skew` [here](https://github.com/mzegar/node-rapids/blob/df-skew-kurtosis/modules/cudf/src/data_frame.ts#L1628), where you were returing `Series<never>`.

But yes, if #240 works, then we can ignore this

Reply:
ah you're right, I'll double check this tomorrow. For whatever reason `ts` was inferring it as `never`, I'll look into this.

Reply:
Verified that this returns `never`.